src/main/java/avd/inf/jdm/rentmycar/RentMyCarApplication.java

```diff
@@ -23,7 +24,7 @@ public static void main(String[] args) {
         SpringApplication.run(RentMyCarApplication.class, args);
     }
 
-
+    @Profile("!prod")
```

Comment:
Nice!
Hoe weet het systeem welk profiel de huidige is?

Reply:
in de docker-compose.yml staat deze regel. De docker-compose.yml is een file welke de server opstart en zo. 
En alleen op de server wordt dit commando dus  meegegeven, dus dan draait bovenstaande code niet (!prod) :)

lokaal voor je dev omgeving is er geen wijziging qua werkwijze :) .

 environment:
      - "SPRING_PROFILES_ACTIVE=prod"